src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -182,6 +187,34 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         return false;
     }
 
+    // Method Description:
+    // - Creates a new profile and navigates to it in the Settings UI
+    // Arguments:
+    // - profileGuid: the guid of the profile we want to duplicate,
+    //                can be empty to indicate that we should create a fresh profile
+    void MainPage::_AddProfileHandler(winrt::guid profileGuid)
+    {
+        uint32_t insertIndex;
+        auto selectedItem{ SettingsNav().SelectedItem() };
+        auto menuItems{ SettingsNav().MenuItems() };

```

Comment:
const?

Reply:
const doesn't do meaningful things for winrt types.